serialization-avro/src/main/avro/giftcard.command.IssueCard.avsc

```diff
@@ -0,0 +1,19 @@
+{
+    "type": "record",
+    "namespace": "io.axoniq.dev.samples.serializationavro.api",
+    "name": "IssueCardCommand",
+    "doc": "Issues a new gift card.",
+    "revision": "1",
+    "javaAnnotation": "org.axonframework.serialization.Revision(\"1\")",

```

Comment:
Curious: why have the `@Revision` annotation on all the messages? I would guess to validate if it works?

Reply:
It demonstrates how to put Java annotations into generated code and I personally like the idea of providing explicit versions to all messages.